paraview/src/CloudberryDemo.js

```diff
@@ -0,0 +1,441 @@
+/* global document */
+import CompositeClosureHelper from 'paraviewweb/src/Common/Core/CompositeClosureHelper';
+import FieldProvider from 'paraviewweb/src/InfoViz/Core/FieldProvider';
+import LegendProvider from 'paraviewweb/src/InfoViz/Core/LegendProvider';
+import Histogram1DProvider from 'paraviewweb/src/InfoViz/Core/Histogram1DProvider';
+import HistogramBinHoverProvider from 'paraviewweb/src/InfoViz/Core/HistogramBinHoverProvider';
+import ScoresProvider from 'paraviewweb/src/InfoViz/Core/ScoresProvider';
+import SelectionProvider from 'paraviewweb/src/InfoViz/Core/SelectionProvider';
+import HistogramSelector from 'paraviewweb/src/InfoViz/Native/HistogramSelector';
+import FieldSelector from 'paraviewweb/src/InfoViz/Native/FieldSelector';
+
+var JSONbig = require('json-bigint');
+
+var httpProxy = "proxy.php";
+var cloudberryHost = "ws://localhost:9000/ws"
```

Comment:
I think we can add a comment to remind the user to specify this is the line to change.